### DISCUSSION TITLE
Restricted Access Mode

### DISCUSSION THREAD

**Discussion by indo2042** [LBPUnion/ProjectLighthouse#692] [Ideas]
It would be nice to have a config option to have users be locked in a "read-only" state if they do not have an email linked to their account. These read-only users would not able to publish levels, comments, reviews etc.

**Comment by m88youngling** [comment]
This would be a good way to force people to set their email and get access to the website, especially if in the future we implement some kind of moderation notification system on the website that informs users of moderation action taken against them.

**Reply by sudokoko** [reply]
+1 - I would not oppose this, but have it be configurable definitely.
